contracts/TransferHelper.sol

```diff
@@ -31,10 +31,6 @@ library TransferHelper {
     (bool success, bytes memory returndata) = token.call(
       abi.encodeWithSelector(0x23b872dd, from, to, value)
     );
-    Address.verifyCallResult(
-      success,
-      returndata,
-      'TransferHelper::transferFrom: transferFrom failed'
-    );
+    Address.verifyCallResult(success, returndata);
```

Comment:
This is because of a change shown [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/61117c4db8497ba489d5e1e127565a011ed6907a/contracts/utils/Address.sol#L135C3-L141C6) in Address.sol openzeppelin contract that we are using